docs/src/content/docs/introduction/op-flow.mdx

```diff
@@ -5,20 +5,21 @@ title: Open Payments flow
 import {
   MermaidWrapper,
   Mermaid,
-  LinkOut
+  LinkOut,
+  Tooltip
 } from '@interledger/docs-design-system'
 import Interactive from '/src/partials/diagram-interactive-grant.mdx'
 
 This page describes each step in the Open Payments flow at a high level. The sequence diagrams are for illustrative purposes and might not be an exact representation of a given flow.
 
 ### Assumptions
 
-- The client's end-user is the payer.
-- The client already has the payer's Open Payments-enabled account details and is able to send payments on behalf of the payer.
+- The <Tooltip content="A mobile or web application or service" client:load>client's</Tooltip> user is the sender.
```

Comment:
Tooltip: **Such as** a mobile or web application or service

Be clear it's an example